modules/custom/moj_resources/src/CategoryFeaturedContentApiClass.php

```diff
@@ -83,53 +83,53 @@ protected function translateNode(NodeInterface $node)
     return $node->hasTranslation($this->lang) ? $node->getTranslation($this->lang) : $node;
   }
   /**
-   * Get nids
+   * Get node ids
    *
    * @return void
    */
   protected function getFeaturedContentNodeIds($category, $number, $prison = 0)
   {
     $series = $this->promotedSeries($category, $prison);
     $nodes = $this->promotedNodes($category, $number, $prison);
```

Comment:
In this context it'd make more sense to rename node to content as that's what it's pulling back.